pureskillgg_dsdk/tome/curator.py

```diff
@@ -279,10 +283,8 @@ def get_match_by_index(
 
         df_header = loader.get_dataframe()
 
-        full_path = df_header["ds_path"][index]
-        key = df_header["key"][index]
-        root_path = full_path.split(key)[0]
-        manifest_key = os.sep.join([key, self._ds_type])
+        root_path = df_header["root_path"][index]
```

Comment:
We are moving away from putting the root path inside the tomes and depending on the config, right?

Reply:
yee

Reply:
@billfreeman44 I think you missed this one? Can you search for any other use of `root_path` and check it's not trying to pull it out of a data frame or json file?

Reply:
Whups.  I can't find any more but I added a test for the single match reader.